driver/flambda_backend_args.ml

```diff
@@ -868,6 +886,11 @@ module Extra_params = struct
     | "dump-inlining-paths" -> set' Flambda_backend_flags.dump_inlining_paths
     | "reorder-blocks-random" ->
        set_int_option' Flambda_backend_flags.reorder_blocks_random
+    | "basic-block-sections" ->
+      let options = [ Flambda_backend_flags.basic_block_sections;
+                      Flambda_backend_flags.use_ocamlcfg ] in
+      Compenv.setter ppf (fun b -> b) name options v;
```

Comment:
Does this disable ocamlcfg if you disable basic block sections?  Not sure that's right.